### DISCUSSION TITLE
TS101, Pinecil v2, 140W, short tips, cables...

### DISCUSSION THREAD

**Discussion by sorgelig** [Ralim/IronOS#1664] [General]
Hello!

I've got both TS101 and Pinecil v2. I've also bought 140W Type-C power supply (UGREEN).

So, here are my observations:

Good:
1) both TS101 and Pinecil v2 work with 28V 5A (140W).
2) TS101 works with both original Miniware long tips and short Pinecil tips.
3) heating speeds of TS101 and Pinecil v2 are same using same tip. ~2-3 seconds using short tip.
4) Heating with long tips on 140W is quite fast too, so may be you don't need short tips.

Bad:
1) short tips have serious heat creep problem. After several seconds short tip becomes pretty hot near the iron body and inside too. May be because it's short or may be because of different metal used. I think it's not a good "feature" since it will heat-up the body and internal components.
2) for 140W you need a special 140W rated cable which is pretty hard cable. It's not comfortable to use the iron with such cable. Both silicon cables from Miniware and pinecil are NOT 140W and both irons show only 20V with those cables.

Overall, TS101 i think is better than Pinecil v2 because of more convenient buttons placement (especially minus button!), improved handle, tips hot-swap, larger screen. Firmware of TS101 is basically on-par with IronOS, i like how TS101 displays temperature and power. Profiles are also convenient. I don't feel i want to change firmware in TS101. However, IronOS handles non-original tips temperature better than Miniware firmware (my observation on TS100).

**Comment by River-Mochi** [comment]
> 2\. for 140W you need a special 140W rated cable


I would not buy any cables that say only 140W rated, they do not follow the USB-IF spec.
Real **PD3.1 usb-c cables are 240W-48v-5amp** rated and  meet the EPR  spec.  I do see people selling "140w" cables online but it's not following any official spec USB-IF spec.   Even the 100w cables have been deprecated by USB-IF org as they want everyone to move to  240W-48V EPR cable (completely backwards compatible with lower chargers and devices).     
The spec for the cable and the chargers should not be confused.   The chargers on the other hand  are harder to make so the first round that came out in 2022  are  140W-28V-5amp  chargers.  next phase will be 36v, then 48v in several years.  However, the max  rated  cable  240W-48v-5amp  is already out and is the only cable that USB-IF  certifies in the EPR category. all other cables that are not 240w are rated as SPR (standard power). 

- I found some very soft and flexible PD3.1, 240W cables.  This one is soft, flexible but is not silicone, it's just TPE https://www.aliexpress.us/item/3256804889230539.html (need be careful on aliexpress, they like to use word liquid silicone for anything that is flexible).  cable works and flexible.
- I'm just as comfortable using the 240W  flexible cables I have for soldering as all my other usb-c cables.
- I have not  tested this one yet but it claims to be a flexible silicone 240w cable. if someone bought this and tested pls let me know. It's $17 and I already have at least 8-9  240w cables.   **https://www.aliexpress.us/item/3256804647643896.html**
- key is to get a cable that is not braided as they are too stiff. look for ones that are smooth casing.
- for most flex, stick to a 240w PD3.1 cable are only **slower USB2.0 data speed** (480mb/s).  Any of the PD3.1 cables that have  20G or 40G  data speed will be too thick and stiff because they have a lot more wires inside to do the faster data speed which  is not needed.   480mbps (usb 2.0) is fast enough for firmware upgrades. How do I know - I bought all of these kinds of 240W 40G, 20G, and several normal  240w/480 ones. and I was mailed several cables as well to test. I bought a few last year too when they were first on market and cost $35-$50 and originally paid more for USB certified ones. now only $10 is cheap for a 240w cable that is not certified - several still work perfectly fine. 
- I don't have any issue with excessive heat using short tips.  I also use  lower temperature solders.  I use both short and long tips with Pinecil V2.  the only time the front handle got too hot was when I was doing  extreme performance testing which  was unusual since I was pushing Pinecil up to 110w  using  a tip in a cup of water.  this is not how it would normally be used.
- in normal use  its fine.
- [video here](https://www.youtube.com/watch?v=nTC-ah4f0hg) with some simple speed tests done to compare  USB-C 20V and 28V chargers.  There is only a few seconds difference so best bang for buck I still recommend people get USB-C 20V charger. but  if you are like me,  I wanted to try bleeding edge charger and got the Apple EPR 140W  in Spring of 2022 ($110).  now there are a lot more choices. 
- fyi,  the reason why miniware (mw)  ts101 even has PD3.1 support is because of Ralim.  Their original release ts101 only did USB-C 20V.  Since he has a library for Pinecil V2 that has supported PD3.1 28V since  July of 2022, he actually personally helped MW  get the ts101 working for EPR PD3.1 chargers 28v ( [reference](https://github.com/Ralim/IronOS/issues/1420#issuecomment-1345234437)).

The Pinecil V2  is more mature for using EPR power at 28V since it has been doing it since July2022 and IronOS was made for it.   The mw new firmware support for it only came more recently (iir ~dec 2022)  after Ralim's helped them with their firmware.  There were several  issues originally with  EPR chargers and pinecil at first, which Ralim and several of us  worked on testing and sorting out. Mainly by spending a lot of money buying a lot of these new EPR PD3.1 chargers. 

 MW benefits from all the early testing done on Pinecil V2 back in the Summer of 2022.  There were specific comm issues that had to be resolved between all the new EPR chargers on the market and the PD  implementation Ralim  is using. 
 
 The Original  PD3.1 libraries that Ralim is using came  from a lot of work by a Pine64 member named Not_Daniel.   Because not_daniel wrote the libraries and agreed share them on Open License, then Ralim  was able to put PD 3.1 support in Pinecil. and ultimately  why  Miniware can now do PD3.1 support for ts101.  
Miniware and ts101 all have a **Pine64 member** dedicated to open source software to thank for the history that they can use 28v EPR charger on the iron (even though Miniware firmware is  proprietary and closed). 

- another reason I prefer the Pinecil V2  is the schematics are also open. you can download and look at the schematic and datasheets for the parts.  Unlike Miniware where  there are no shared schematics and the firmware is closed.  If you look at the thread  for asking if IronOS can be ported to  ts101, you will see that MW refuses to share the schematic,  and therefore it would take many weeks of backwards engineering to try and make a different firmware for it.
https://github.com/Ralim/IronOS/issues/1420#issuecomment-1404510840

-  unless there are some huge feature differences on some product,  I like to support the open firmware and hardware devices that share the schematics whenever I can.

**Comment by sorgelig** [comment]
About cables: It doesn't matter how stiff or flexible cable is, for PD3.1 cable has to have e-marker chip where 5A 28V mode is written. Without that chip, PSU won't enable 28V. First one is not available anymore. Second one has e-marker and looks the one which will work. But it's only 1m which is quite short (at least for my setup). And it's quite expensive.

About irons: TS100 was existed very long before Pine64 cloned it. So they basically cloned HW which was well tested before them. So if you want talk this way, then it definitely not to Pine64 benefit. PD3.1 is nothing more than just send a data to PSU asking for 28V. PD3.1 PSU are still not so wide spread. It's pretty new thing, not only for irons, but as PD3.1 itself. Nothing to test there as long as correct data sent to PSU. MOSFETs in that soic-8 package are usually rated at >35V.

I'm software and hardware developer and believe me i know what is open source and community support, but sometimes i like to speak from generic user who just wanna use some device without messing too much with updates. Today we've got used to devices released to market with not well tested firmware/software/driver. We already forgot how it was when you buy a device and simply use without any updates. As a generic user i would prefer to get a device not requiring update. Iron requiring huge community support to be able to solder, i think, isn't a good device. I don't think it's correct when independent developer Ralim solely makes firmware for Pinecil instead of Pine64. Pine64 sells not fully supported Iron relaying on Ralim's further development. This is fundamentally wrong, i think. Now we have v2.21 update which leaves some Pinecil v2 in non-working state. If Pine64 would sell Iron with final firmware supporting all features, then such issue wouldn't happen.
TS101 comes with quality updates over TS100: bigger screen, more convenient handle, hotswap of tips, type-c power. Interface is much improved even over IronOS - it's more clean, less cluttered. It doesn't need update, actually. Update to PD3.1 simply eliminates advantages of Pinecil. Pinecil v2 is identical to v1 by exterior, so Pine64 didn't work hard to bring a really new iron. Basically, the only difference between V2 and V1 is PD3.1 support. Ok, and bluetooth. Next version probably will be able to play MP3.

I have both irons and bought them for my own money. That's my honest and independent review/conclusion.

**Reply by gamelaster** [reply]
> I don't think it's correct when independent developer Ralim solely makes firmware for Pinecil instead of Pine64. Pine64 sells not fully supported Iron relaying on Ralim's further development. This is fundamentally wrong, i think.

Please note, that this is exactly how Pine Store works. Pine Store makes only hardware for developers, to develop on. They do not do software. Pinecil V2 was made together with Pine Store and PINE64 Community (thus also Ralim), and Pine Store support developers, which are working on it.

> We already forgot how it was when you buy a device and simply use without any updates. As a generic user i would prefer to get a device not requiring update. 

Well for sure, you can use another irons, which will never get any update with bug fixes or new features.

Pinecil V2 ships with fully functional firmware, although, not using all new features. v2.21 introduced BLE support for Pinecil V2, and even it was tested properly, it still fails on some devices. But you can still use Pinecil V2 with older firmware, which works just fine, just without new features, improvements, bug fixes, translations etc.

Yes, you are right and I agree with you, that products should arrive with working firmware, which Pinecil V2 was, but I think that receiving updates is good and unique, since you can get more features and fixes for example, 2 year old hardware, instead of buying a new just to get new software features, as some other companies do.

**Reply by River-Mochi** [reply]
> And it's quite expensive.

You were saying the EPR  cable was "hard" cable and hard to use not as comfortable.  I was responding to that.  there are several flexible cables available now on the market  and I gave some hints for how to get one that is easier to work with.   I know bc I have both the thicker ones that don't bend as nicely and the newer more flexible 240w ones that are a joy to solder with.  you also don't need to use EPR cable if you don't like them.  a PD60 or PD100 cable works too.

Price for EPR 240W cables has gone way down in past 3 months I gave links to 2 up top that are more reasonable price (compared to last year when they were $35+, a $15 cable is a bargain). I have several other 240w cables that all work and cost $5-$10, (the cheaper ones are not certified). all of my  240W-48v-5a EPR cables have an e-mark in them otherwise I would not be able to get 28V on Pinecil V2 and they would not pass the PD3.1 test in my PD Analyzer.  

I'm not sure what you mean but  all 240W cables have to have an emark in them; that is part of the USB-IF protocol.  if the cable does not have an emark then  the charger will not pass more than 20V for safety reasons. 
A real silicone cable always cost more to make than TPE or PVC, this is known reason.  Even back when I  was buying lower  PD60 and PD100 cables, real silicone cost me ~$12-$15  unless I got cheap $3.99 one from pine64.com but they charge $12 for shipping so only worth getting $4 cable if buying with other stuff. 

these new cheaper cables (not-certified)  may have clone e-mark in them, but some people don't care if it's certified or not as long as the cable works.   The $35  cable I bought last year is true Certified but it's full 240w-40G  (which is very very fast data speed)  - but this makes cable too thick for soldering and stiff.  Hence I currently recommend people stick with **thinner more flexible**  240w /480mb cables for soldering.  My  240-480 cables are all a lot more flexible than my 240-20G and 240-40G cables and ALL of them have Emark chip and all of them allow 28V from EPR charger to Pinecil V2 and the USB-C pd3.1  tester.  USB 2.0 is plenty fast for firmware updates and thinner cables with USB2.0 will be more flexible (as long as they are not braided which also hard to work with). 

- I don't think EPR charger and cable are really needed,  most people will be happy with 20V speed and save $50. I bought a few EPR chargers but only because of curiosity of new tech.  I did some basic speed tests and 20V is fine.
reference:   https://www.youtube.com/watch?v=nTC-ah4f0hg

> So they basically cloned HW which was well tested before them

For this, I would simply point you to Ralim's reponse here https://github.com/Ralim/IronOS/discussions/1257#discussioncomment-3006001. He was part of the design of Pinecil itself.  You tried to say before in that discussion that pinecil is clone of ts100 but that is not true either.  Before ts100 there were also portable size irons that have the small pen shape. ts100 was not the invention of this at all. Pen shape is a generic  Iron shape that has been around for many years previous to ts100.   the ts101 though took the power design from pinecil to add usb-C  support and although it started with only 20V usb-C suppport,  with ralim's help mw was able to change their firmware to pd3.1 - otherwise all ts101 people would only be able to use max of usb-c 20v. 

> I don't think it's correct when independent developer Ralim solely makes firmware for Pinecil instead of Pine64. Pine64 sells not fully supported Iron relaying on Ralim's further development.

For this it's important to first understand the premise of why Pine64 was founded and why they exist. Trying  to compare them like they are like a normal company - but they are far from how normal companies operate.  Pine64  is basically run more like a non-profit and has low margins on items on purpose, some items have zero margin. Yes that's right **zero** margin - read here:  https://www.pine64.org/2019/08/19/its-time-to-start-giving-back/

The mission of Pine64  is to create hardware specifically  for developers to tinker with and to encourage/support the FOSS community. Their model is completely different from other companies like Miniware and others. 

Changing the issue/topic does not dissuade me  from leaning for hardware that has open schematics and made specifically for the open source community of hobbiest & developers.  The real history is Pinecil was built  from ground up in cooperation with Ralim.
We are all pine64 members.

 IronOS being open means that other volunteers help maintain it even if Ralim gets busy. even very old irons like Pinecil V1 still get updates rather than being forgotten about which usually happens with other companies.

- the other reasonI prefer the Pinecil V2  is that it's easy to repair.  we are given the full schematics by Pine64.  I've blown a mosfets  from pushing too much bench power trying to test how high I can go and other things.  When you don't have schematic and open source community,  old broken items go in the trash to hard to research and fix.
- Pinecil V2  can hot swap if you are using same length of tips.  it's only unplugged if switching to long tips. the ts101 can only use the longer tips.  I  own all 11 styles of ts100 longer tips and I have the 4-pack of pinecil short tips.  I prefer  soldering with short tips whenever I can  because it gives me more control over soldering, hand to pcb distance.  However,  I'm still also using my longer tips because they still work.   if I have a very delicate area to solder and I need more control there, I definitely get advantage holding pinecil with short tip closer down to the work. 

FYI - miniware didn't / doesn't pay Ralim for his help and they are a traditional for-profit company.  He specifically told us this. 

-  Since you wrote this to compare two irons, there are other things left out of the comparizon. Miniware has barely any support page for TS101 compared with the vast knowledge in support wiki for Pinecil. 

https://wiki.pine64.org/wiki/Pinecil

This Pinecil Wiki  has everything about Pinecil, Pinecil History, photos, How to repair, how to modify, how to add LED lights, how to run Bluetooth apps on it and all the side open source projects spun off from it.   When I look at Miniware support page for TS101 - there is hardly anything.  There are people who use the Pinecil for RISC-V dev and don't solder with it- never put a cartridge tip in it.  This can not be done with ts101 because there are no schematics and the firmware is not open.  The reason we have the PD3.1 epr libraries to use is because not_daniel,  a Pine64 member was able to use his pinecil as a risc-v dev unit (no tip installed)  to figure out the usb-c protocol and make the libraries.  This continues on the V2 also.  this can not be done on a device like ts101 because it's not open. 

- Pinecil is not sold as just a sodering iron like Miniware ts101.   It is fully intended to be a RISC-V  development device for hobbiest.   
<img src="https://user-images.githubusercontent.com/97197236/233759651-0f4e71c1-1f5f-4857-badf-ed7fac08d2e3.png" >

- One of my favorite things about the Pinecil V2  is the PineSAM  app that I use on my phone in a stand while soldering.  I can see temperature on large phone screen in color  and when I tap on phone to change temperature it instantly changes it on the Pinecil V2.   The miniware ts101 can not do this.  the minor small increase in screen size does not help me in the way  Pinecil V2 on any phone or ipad  or  PC/laptop can help me. 
https://builder555.github.io/PineSAM/

- When I solder this is exactly what I see for Pinecil V2 -  I don't even use the  small iron screens anymore. 
- We also have Two Preset buttons on pineSAM where I store  extra set temperatures I want for pinecil.  compares to TS101,  I like this better.   This is only possible in Pinecil V2 and not in TS101. 
- and if someone doesn't like the colors of PineSAM  - they can just fork the code and make all their own customizations.
- I can also change all pinecil settings using pinesam, hence another reason why small screen does not impress me.
-  I can see live temperature of pinecil on my phone from across the room (or pc/laptop).  I prefer the phone though bc I can tap on the buttons with gloves on hands while soldering.
-  Pine64  gave  us extra  features on the chip and the PCB  so that DIY tinkerers and developers could take  dev to next level compared to locked devices which you only get exactly what the company gives you. 

This is exactly what I see on my android phone which I use for Pinecil V2 to totally control temperature and change all settings.  I grabbed my last year phone and it sits in work room table now instead of dead in a drawer.
<img src="https://user-images.githubusercontent.com/97197236/233759909-ec6af8b6-51f1-4a5d-b44c-8cf78a5fe80f.png" width ="300">

- imho, this screen size  for accessibility is so much better than just a tiny little screen size on the iron.  I can make PineSAM as big as I want, even blow it up on a 27" monitor if I want since it runs in any browser. This would be helpful if teaching a Maker class tutorial. 
- I also like that I could be part of the design process for PineSAM to work better with Pinecil and IronOS and I am working with great volunteers on that to bring things to next level.  sure some company might come along next year  and make an iron that you can run from  a phone (and they might even take pinesam and ironOS code to do it too),  part of the fun is being part of discovering and making things work as tinkerers and hobbiest. 

-Pinecil V2 also has  a pcb that designed with an empty footprint on it that lets me add a hall effect sensor. miniware ts101 doesn't have this ability.  this means if people want to enahance their Pinecil hardware even more,  there are features they can DIY.  It's not put on Pinecil to keep cost down, but they leave  footprint there for anyone who wants to add the chip or  wants to dev that further.  This is all in the spirit and purpose of Pine64.

I'm leaving out a lot more details and reasons here why I prefer the  Pinecil v2  as a portable soldering iron since I've already given many reasons.  This was put in discussions so it's open to people saying also why pinecil V2 has many benefits over ts101,  and not just one person point of view.  details were left out about the extra things that come with Pinecil V2.  

If I was to consider something else, it would be an open source Station - which doesn't really exist yet. 

### One of the best reasons to  get a Pinecil V2 is the huge community of Pine64 members who are very active at all levels of the process whether it's coding, testing, or writing articles.

Pine64 members live all over the world, and therefore active 24/7. 

I don't work for pine64 either, all of us are volunteers and this is my honest opinion.

here is one of the receipts for my purchase, I also bought stuff last year.  tbh, I did look at buying a ts101 too when it was released but I didn't see purpose for such a small screen size increase when I can see Pinecil V2  and control temperature on a phone or browser which is bigger with color.   I can do that with Joric's web api without installing anything on my PC or phone.  for more advanced features I use PineSAM.  both are color large screen- any size.  I truly believe for accessibility, bluetooth  and/or wifi  on phone/PC screen  is better flexibility and choices over trying to make screen larger on the hand held part.  I'd  like the handle part to be smaller not bigger,  and use a station  to have all the data if working indoors.  if bigger then perhaps back half could be flat and wider and front half be skinnier.

<img src="https://user-images.githubusercontent.com/97197236/233762371-55d552bc-a961-4720-9ea5-794c20211b4d.png" width="400">

**Comment by sorgelig** [comment]
* Regardless existence of small soldering irons before TS100, it WAS invention. TS100 was the first useful pen-sized iron. Other irons of that time with similar size were more or less rubbish/toys useless for real soldering. If you will search in youtube, 90% videos about pen-sized soldering irons will be about TS100 (ok, after appearance of Pinecil it started to share that percentage). So TS100 was a special, and that's why it was cloned by Pinecil. I'm not talking about schematics - it doesn't matter. They cloned the outlook with subtle change, they cloned the tips from Miniware - this is why they are 100% compatible to each other. Which cpu they used or specific resistors or transistors - it absolutely doesn't matter.
* You forgot that Iron is made for a sole purpose - to solder, not to play with it as with smartphone. If it works well then you don't need to know even how to flash it. 99% features of Pinecil you mentioned is to leverage the problems. No problems - no 99% of advantages. Good Iron is when you plug it and start to solder. If it gets broken after 2 years, just buy a new one. There is no point to repair it.
* BT is still useless to me. At first - it simply doesn't work in my PInecil v2 :) Anyway, i didn't expect to use it. Why i need to use smartphone while i can simply press a button on iron if i want to change the temperature? I don't even need to put iron back to stand. Why i have to put my iron to stand, put pcb to table and then take smartphone, possible wake it up, unlock just to change the temperature? This is silliest use case!
* TS101 does support short tips just fine.
* I agree PD3.1 is mostly a gimmick feature. 20V is a sweet point where you can get cheap silicone cable and generic power supply. So, basically with TS101 you don't even need to update FW. Just buy it and use. I've bought 140W PSU with cable just for curiosity. May be half year later 250W silicone cables will cost $3 and i will switch to that setup.

> I truly believe for accessibility, bluetooth and/or wifi on phone/PC screen is better flexibility and choices over trying to make screen larger on the hand held part. I'd like the handle part to be smaller not bigger

This and the fact you don't have TS101 make doubt in your honesty. You prefer a smaller screen than big if it even doesn't change the size of device? It's so honest :)

**Reply by River-Mochi** [reply]
> This and the fact you don't have TS101 make doubt in your honesty. You prefer a smaller screen than big if it even doesn't change the size of device? It's so honest :)

**There is no need to personally attack people and question their honesty.  It is an ungraceful way to be dismissive of other people.** 

It does not help in a discussion about personal experiences with a soldering iron. I never questioned your honesty so don't question mine. stating my experience just like you did.   If you don't like my personal experience or disagree, it's still not right to question my honesty as that is a personal attack.  if you did not want other people to write their experience then why post this in open discussion format, many options if people don't want responses to their personal opinion statements, like a personal blog/webpage where no comments are allowed. 

I'm not sure exactly what specific things you are implying im not honest about...

- are you saying I never used PineSAM on a phone? I can link videos as proof if needed of using Pinecil with PineSAM and how it helps my accessibility. I think I might already have short videos up of this.
- is there some honestly issue my statement that Miniware doesn't share schematics?  If you have link to ts101 schematics and also their detailed support pages I'm happy  to look and be corrected on this item. 
- questioning my honesty is vague and troll way to dismiss other peoples experiences because they are not the same as yours. my personal experience with the Iron is mine - no reason to call people dishonest because it makes no sense.
- one example, I don't have a problem with short tips, and you  have issue with short tip getting hottoer. but I never questioned honestly of your experience .  other people might use higher temps. this is my experience. 
- people will respond to open discussions in various ways, all with their slice of experience whatever that might be.  trolling and turning it into personal attack only shuts down discussion.

**Comment by sorgelig** [comment]
Pinecil has small corners in the front intended to be a mini stand while laying on flat surface. But either there were 2 designers or may be one but lazy one. At later point they decided to add a rubber wrap, but forgot about that mini-stand.. It's not flat anymore.. That's how Pine64 careful about details..

**Reply by gamelaster** [reply]
Note that Pinecil V2 was made altogether with community, everyone could give a feedback. That's why many bugs  were fixed in V2, and new features implement based on suggestion from community. If you have any suggestions or recommendations for future versions, let us know in PINE64 community chats. (you can find links at wiki.pine64.org )

**Reply by sorgelig** [reply]
Don't take me wrong, i don't deny community support and feedback, but soldering iron task is pretty much simple and it's not a problem to make a solid firmware not requiring future updates (well, unless some really new features added). Pine64 stepped back from firmware. They think if they open schematics, then they are free from making a solid firmware. This is wrong. They sell hardware claiming BLE support while stock firmware has no BLE support at all. And when new firmware (3rd party, btw) v2.21 was released, it turned out some Pinecil v2 stop to work.

> That's why many bugs were fixed in V2

Many bugs in stock firmware is a bad thing. Reputable company should not release device with many bugs. Soldering iron is not smartphone. It's just heating device - everything (well, almost) should be fixed BEFORE release, not after.

**Reply by gamelaster** [reply]
> and it's not a problem to make a solid firmware not requiring future updates

It is, as Pine Store does not do any software. Pine Store does only and only, hardware, and handles manufacturing, shipping, etc.

Software is all in hands of community. Take it as when you are buying device from Pine Store, you are purchasing only hardware.

> They sell hardware claiming BLE support 

When you take look on Pine Store's Pinecil listing, there is no BLE mentioned at all. Although, anyway, Pine Store sells hardware, so in terms of hardware, it supports BLE.

> Many bugs in stock firmware is a bad thing. Reputable company should not release device with many bugs. Soldering iron is not smartphone. It's just heating device - everything (well, almost) should be fixed BEFORE release, not after.

Sorry, didn't said exactly, I meant hardware bugs were fixed.
Software is in charge of community, and I like to rather have opportunity to receive updates, than getting proprietary device, and having no ability to get bugs fixed, because there always be.

**Comment by River-Mochi** [comment]
Pine store picked the BL706 chip. it just happens to have  Bluetooth LE  capabilities which are not needed for purpose of soldering .   One reason they picked it is because it is readily available. but it now gives us not only RISC-V   opportunity to develope for fun but also  bluetooth hidden inside. 

Have to remember that Miniware sells their irons to use as soldering iron and nothing else.  the Pine64 pinecil is a different animal.  it's also a RISC-V  development platform which is why pine64 has to give us all the schematics. otherwise no open source  fun and mayhem can happen. 

To be clear other soldering irons are sold with firmware and do one job - solder.
Pine64 pinecil is sold as a development tool for open source community that just happens to also be able to solder pretty well. 

Pine Store  decided to go with this chip  but doesn't support BLE  capabilities and never claimed to.  they go with chips like this for the same reason  they went with a RISC-V chip.  It has hidden features  for developers who want to tinker with it  and  explore it and open that up. 

Bluetooth capabilities are 100%  on the community to unlock should  people want to explore this.  That is the beauty of Pine64, they don't lock things down.  it's an open engine and you can tune it and alter it whatever way you like - also why it's only $26+$12 or about $38 for many countries (ymmv depending on country). 

you can't do that with miniware ts101  because it's locked down, no schematics, no access to firmware. 

you keep attacking this without  giving counter information to show where and how Miniware provides these dev capabilities in the ts101?  what hidden features can be unlocked with ts101?  is there bluetooth  in the chip that devs can tinker with?  a schematic that lets them even start?

I think it might be worth me repeating that the Pinecil  based on the mission of pine64 is not simply just a soldering iron.  When they decide on chips and other components  they "also"  think about what cool things  would be nice for open source community to explore. 
So they went with BL706 chip.  because they know  hackers/tinkerers/devs out there will jump on it for hobby, self-learning and fun.
![image](https://user-images.githubusercontent.com/97197236/233815816-f70f29e1-970f-41e0-bd8f-f023c75bfeec.png)


How does Miniware do this?  if they don't do this, it's like comparing apples and a truck.  The two irons serve different purposes, but both can solder.  
One is made  with input of open dev community for hardware they would like to see and play with. 

Fun fact:  I remember the community chats where we were all joking around saying we would like Bluetooth LE or Wifi  on Pinecil and weird scenarios people came up with.  And Ralim and others saw this, so when  it came time to pick between different new chips, they took the jokes and comments from community into consideration. And gave us a chip we could unlock and explore  which is the bluetooth under the hood for DIY  exploration.
